Mathlib/Analysis/NormedSpace/OperatorNorm/Basic.lean

```diff
@@ -281,6 +280,16 @@ theorem opNorm_smul_le {𝕜' : Type*} [NormedField 𝕜'] [NormedSpace 𝕜' F]
     gcongr
     apply le_opNorm
 
+/-- A continuous linear map is `M`-Lipschitz if its operator norm is at most `M`. -/
+theorem lipschitzWith_of_opNorm_le {M : ℝ≥0} {φ : E →SL[σ₁₂] F}
+  (h : ‖φ‖ ≤ M) :
+    LipschitzWith M φ := by
```

Comment:
Can you just upgrade [ContinuousLinearMap.opNorm_le_of_lipschitz](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/NormedSpace/OperatorNorm/Basic.html#ContinuousLinearMap.opNorm_le_of_lipschitz) to an `↔`, and then re-add that result and this one via the `alias` command? Also, the proof of this direction should just be something like (untested): `φ.lipschitz.weaken h`.